Jenkinsfile

```diff
@@ -97,6 +97,10 @@ def fast_tests_stage(Platform platform, Configuration config)
     stage("Unit Tests ${stage_name_suffix(platform, config)}") {
       sh "./scripts/ci-tool.py -T ${config.label}"
     }
+
+    stage("Etch Lang Tests ${stage_name_suffix(platform, config)}") {
```

Comment:
This will make the tests run on all platforms including Mac, and on all parallel PR merge builds. I'm not opposed; just checking this is what you were going for.

Reply:
That was sort of what I was expected. I didn't really consider the Mac. They are very short set of tests to I tend to think let's keep them and then move them if they start to take a long time.